analysis/webservice/algorithms/doms/DomsInitialization.py

```diff
@@ -49,6 +49,7 @@ def init(self, config):
         log.info("Cassandra Protocol Version: %s" % (cassVersion))
 
         dc_policy = DCAwareRoundRobinPolicy(cassDatacenter)
+        #dc_policy = WhiteListRoundRobinPolicy([cassHost])
```

Comment:
Could you delete this commented line and the unused import in this file?

Reply:
ok, I did that.